src/Controller/CommentsController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the BT project.
+ *
+ * Copyright (c) 2023 BT
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Controller;
+
+use App\Entity\Article;
+use App\Entity\Comments;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class CommentsController extends AbstractController
+{
+    #[Route('/delete-comment/{id}', name: 'delete_comment')]
+    public function delete(EntityManagerInterface $entityManager, Comments $comment): Response
+    {
+        $loggedInUser = $this->getUser();
+        $article = $entityManager->getRepository(Article::class)->findOneBy(['id' => $comment->getArticle()->getId()]);
```

Comment:
Please move this in the IF, like this you avoid a databas query in case it's not the right user